Formula/gst-plugins-bad.rb

```diff
@@ -40,6 +40,10 @@ class GstPluginsBad < Formula
   depends_on "srtp" => :optional
   depends_on "libvo-aacenc" => :optional
 
+  # gst-libs/gst/gl/cocoa/gstglwindow_cocoa.h is missing in the 1.12.1 release. Fixed by this commit:
+  # https://cgit.freedesktop.org/gstreamer/gst-plugins-bad/commit/?id=450b1c92abe2ae8f84e557f61c9512a1f4006bab
+  patch :DATA
```

Comment:
Now you can use this url:
https://raw.githubusercontent.com/Homebrew/formula-patches/4fba405/gst-plugins-bad/gstglwindow_cocoa.patch